modules/test/maven-executor/src/main/java/com/liferay/maven/executor/MavenExecutor.java

```diff
@@ -139,6 +139,10 @@ public Path getMavenLocalRepositoryDirPath() {
 		return Paths.get(dirName);
 	}
 
+	public String getRepositoryLocalUrl() {
+		return System.getProperty("repository.local.url");
```

Comment:
Shouldn't Maven Executor use this property somehow? Otherwise, it's pointless to keep it here. For instance, shouldn't it configure Maven to point to that repo as well?